crates/revm/src/instructions.rs

```diff
@@ -74,175 +72,165 @@ pub enum Return {
     CreateContractWithEF,
 }
 
+pub fn return_stop(interpreter: &mut Interpreter, _host: &mut dyn Host) {
+    interpreter.instruction_result = Return::Stop;
+}
+pub fn return_invalid(interpreter: &mut Interpreter, _host: &mut dyn Host) {
+    interpreter.instruction_result = Return::InvalidOpcode;
+}
+
+pub fn return_not_found(interpreter: &mut Interpreter, _host: &mut dyn Host) {
+    interpreter.instruction_result = Return::OpcodeNotFound;
+}
+
 #[inline(always)]
-pub fn eval<H: Host, S: Spec>(opcode: u8, interp: &mut Interpreter, host: &mut H) -> Return {
```

Comment:
Why no return?

Reply:
Not needed, `interpreter_result` is changed only when loop needs to stop, for most instructions it was same `Result::Continue`. i didn't get big perf by removing this but it is cleaner.